lemur/plugins/lemur_verisign/plugin.py

```diff
@@ -221,8 +221,12 @@ def create_certificate(self, csr, issuer_options):
                 extra={"common_name": issuer_options.get("common_name", "")}
             )
             raise Exception(f"Error with Verisign: {response.content}")
-        # TODO add external id
-        return cert, current_app.config.get("VERISIGN_INTERMEDIATE"), None
+        # DONE: TODO add external id
+        authority = issuer_options.get("authority").name.upper()
+        cert = response_dict['Response']['Certificate']
+        external_id = response_dict['Response']['Transaction_ID']
```

Comment:
I haven't checked the response recently, but just in case there is a chance that no Transaction_ID is added to the response, we could maybe handle it this way:

```
external_id = None
if 'Transaction_ID' in response_dict['Response'].keys():
    external_id = response_dict['Response']['Transaction_ID']
```